client/a8c-for-agencies/sections/signup/signup-v2/components/multi-step-form/blueprint-form/index.tsx

```diff
@@ -0,0 +1,180 @@
+import { Button } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import Form from 'calypso/a8c-for-agencies/components/form';
+import FormField from 'calypso/a8c-for-agencies/components/form/field';
+import FormRadio from 'calypso/components/forms/form-radio';
+import FormTextarea from 'calypso/components/forms/form-textarea';
+import './style.scss';
```

Comment:
Can we add an extra line before `import './style.scss';`. We have this throughout the app.


```suggestion

import './style.scss';
```